lib/hca/service.rb

```diff
@@ -3,10 +3,12 @@
 require 'common/client/base'
 require 'hca/enrollment_system'
 require 'hca/configuration'
+require 'va_1010_forms/service_utils'
 
 module HCA
   class Service < Common::Client::Base
     include Common::Client::Concerns::Monitoring
```

Comment:
you should add this line and the `STATSD_KEY_PREFIX` to the EZR service, it's required for logging stats to datadog

Reply:
Should I use the same naming convention as the hca const? AKA:
```
STATSD_KEY_PREFIX = 'api.1010ezr'
```

Reply:
sure, you can give it any unique name

Reply:
Done!